cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -155,12 +156,13 @@ public void run() {
                         dismiss();
```

Comment:
This dismiss() shouldn't run if the popup has been dismissed (it won't be showing and we check isShowing()). Are you sure the problem is in this library and not in the way the test that fails displays the coachmark?